### DISCUSSION TITLE
Adguard cannot be update anymore

### DISCUSSION THREAD

**Discussion by tomdh76** [tteck/Proxmox#691] [Help Needed] [answered]
I used the adguard update button on the browser but for a couple of releases this button does not work anymore. I am stuck right now at version v0.107.14. Any ideas how to update?

**Comment by tteck** [comment]
I'm on v0.107.16
Do you get any errors?

**Reply by tomdh76** [reply]
If I click on the button "update now"on the top I get on the right side a red box with "Auto-update failed. Please [follow these steps](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#manual-update) to update manually."
Are there any logs I could check?

**Reply by tteck** [reply]
It seems that I remember having to do that. Not sure though. Just follow those instructions.

**Reply by btehan** [reply]
Worked for me :-)

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/711#discussioncomment-4099516

**Comment by ktfcaptain** [comment]
> If I click on the button "update now"on the top I get on the right side a red box with "Auto-update failed. Please [follow these steps](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#manual-update) to update manually." Are there any logs I could check?

I have reinstalled AdGuard so many times because of this issue, which was never a big deal....but I just figured out my really dumb issue so thought I'd share...

My default 2GB storage was full from previous updates.......I feel really dumb.

I went to manually update it and saw the files there, deleted them all, then ran the update from the webui.  I don't know if they're supposed to delete on their own or how it happens but I just increased the storage so it'll take longer and I'll forget about this then run in circles again.....

**Reply by tteck** [reply, marked as answer]
If you follow the update instructions, it will remove those files after the update.

![Screenshot 2023-11-23 10 37 10 PM](https://github.com/tteck/Proxmox/assets/21342670/05fa9caa-ec43-4982-8e36-25cf8d79778f)

The command `update` only works on LXCs created after Oct. 5
